model-server/src/main/kotlin/org/modelix/model/server/handlers/RepositoryOverview.kt

```diff
@@ -27,21 +27,27 @@ import org.modelix.model.server.templates.PageWithMenuBar
 
 class RepositoryOverview(private val repoManager: IRepositoriesManager) {
 
+    class RepositoryIdWithBranchNames(val repositoryId: String, val branchNames: List<String>)
```

Comment:
In IRepositoriesManager, branches are sets. Is order important or could we limit this to a set, too?

Reply:
Done